### DISCUSSION TITLE
保存文件的警告

### DISCUSSION THREAD

**Discussion by jackydi** [tamlok/vnote#2248] [Q&A]
昨天刚刚升级到3.15，今天在编写文档的时候，总是跳出下面的警告，这是怎么回事？如何解决？正在编辑的时候就会跳出来

![image](https://user-images.githubusercontent.com/50565266/191194269-05f63185-f8d7-4a5f-9d4f-8247dbda697e.png)

**Comment by jackydi** [comment]
在编写文档的时候，这个对话框隔几分钟就会跳出来一次，虽然按一下回车键就可以关闭，但经常就把要打的字弄没了，很是烦人，能不能不让他总是跳出来？

**Reply by tamlok** [reply]
是不是有同步软件在外面修改文件？

**Reply by jackydi** [reply]
用坚果云同步了这个目录，但应该只是读取吧，它还会修改这个文件吗？

**Reply by tamlok** [reply]
那就不清楚坚果云了呀，你试着暂停同步用一段时间。

**Reply by jackydi** [reply]
我不用自动保存了，手动保存，目前没有问题。

**Reply by tamlok** [reply]
难道是自动保存有bug。。。

**Comment by jackydi** [comment]
补充说明一下，设置中的保存模式是“自动保存”。
